.eslintrc

```diff
@@ -50,7 +50,7 @@
 /**
  * Possible errors
  */
-    "comma-dangle": [2, "always"],    // http://eslint.org/docs/rules/comma-dangle
```

Comment:
I'm 👍  on these ESLint changes, but do you mind breaking them out into a separate PR?